tensorflow/contrib/learn/python/learn/utils/export_test.py

```diff
@@ -63,32 +63,33 @@ def _get_default_signature(self, export_meta_filename):
       signatures_any[0].Unpack(signatures)
       default_signature = signatures.default_signature
       return default_signature
-
+  
```

Comment:
I think the linter will complain about the added space here and on L85... delete it, please.